jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/security/PropertyUserStore.java

```diff
@@ -166,7 +149,8 @@ protected void loadUsers() throws IOException
         Properties properties = new Properties();
         try (InputStream inputStream = config.newInputStream())
         {
-            properties.load(inputStream);
+            if (inputStream != null)
+                properties.load(inputStream);
```

Comment:
I think you should add an else branch that throws `new IllegalStateException("Config cannot be read: " + config)` or something similar.